forms-flow-data-analysis-api/Dockerfile

```diff
@@ -13,7 +13,7 @@ ADD . /forms-flow-data-analysis-api/app
 RUN pip install .
 
 EXPOSE 5001
-RUN python3 -m nltk.downloader punkt vader_lexicon subjectivity
+RUN python3 -c "from transformers import pipeline; pipeline('sentiment-analysis', model='xaqren/sentiment_analysis')"
```

Comment:
Please make the same change in jobs Dockerfile too

Reply:
sure